app/views/course/assessment/assessments/show.html.slim

```diff
@@ -24,7 +24,8 @@
         td = format_datetime(@assessment.end_at) if @assessment.end_at.present?
 
   h2 = t('.finish_to_unlock')
-  / TODO: dependencies
+  - @assessment.assessment_conditions.includes(:conditional).each do |condition|
+    = display_conditional_for(condition)
```

Comment:
@allenwq we can do that but the condition partial is used to display the condition only (if this is mission 1, the condition is score at least 60% for mission 1), and nothing to do with the conditional. 

So we still need to somehow display and link the conditional ><